receiver/jaegerreceiver/config.go

```diff
@@ -30,12 +30,19 @@ type RemoteSamplingConfig struct {
 	configgrpc.GRPCClientSettings `mapstructure:",squash"`
 }
 
+type SecureSetting struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
+	// Configures the receiver to use TLS.
+	// The default value is nil, which will cause the receiver to not use TLS.
```

Comment:
I thought we decided that for whether TLS was enabled or not it would be part of the endpoint? Maybe we were discussing more in the context of clients but I think maybe that should also apply to listening receivers so that things are uniform?

Reply:
Created this issue to track creating HTTP*Settings so we can offer a config similar to the gRPC*Settigs